test/lib/bamboo/adapters/test_adapter_test.exs

```diff
@@ -54,6 +54,7 @@ defmodule Bamboo.TestAdapterTest do
     end
 
     sent_email |> TestMailer.deliver_now()
+    assert_email_delivered_with(from: "foo@bar.com")
```

Comment:
It took me a second to understand this test, and I was wondering about the idea of having it be its own (named) test, but this does follow the existing pattern of including all of these within the same test about the helpers, so maybe making changes to it would be a separate issue? Thoughts?

Reply:
I'm not a huge fan of this style, but I wasn't going to fight against the current standard. I don't have a strong opinion in favor of changing it but I would say it's a separate issue.